lib/gui/config-overrides.ts

```diff
@@ -0,0 +1,42 @@
+import makeDebug from 'debug';
+
+import {getRecordModeEnumSafe} from '../server-utils';
+import {TestplaneConfigAdapter} from '../adapters/config/testplane';
+import {ToolName} from '../constants';
+
+const debug = makeDebug('html-reporter:testplane');
+
+export const overrideTestplaneConfig = (toolName: ToolName, config: TestplaneConfigAdapter): void => {
+    if (toolName !== ToolName.Testplane) {
+        debug(`Skipping config overrides because toolName is not Testplane`);
+        return;
+    }
+
+    if (process.env.HTML_REPORTER_NO_CONFIG_OVERRIDES) {
```

Comment:
will we write about this environment variable somewhere in the documentation?

Reply:
Discussed & decided to implement this ability in the UI instead of using env vars over here.